examples/python/graphiti_example.py

```
@@ -2,8 +2,10 @@
 
 import cognee
 import logging
+import structlog
+from cognee.shared.logging_utils import setup_logging
 from cognee.modules.pipelines import Task, run_tasks
-from cognee.shared.utils import setup_logging
+from cognee.shared.logging_utils import setup_logging
```

Ruff (0.8.2)

Redefinition of unused `setup_logging` from line 6

Remove definition: `setup_logging`

(F811)